easybuild/easyconfigs/l/libunwind/libunwind-1.1.eb

```diff
@@ -0,0 +1,24 @@
+easyblock = 'ConfigureMake'
+
+name = "libunwind"
+version = "1.1"
+
+homepage = 'http://www.nongnu.org/libunwind/'
+description = """The primary goal of libunwind is to define a portable and efficient C programming interface
+ (API) to determine the call-chain of a program. The API additionally provides the means to manipulate the
+ preserved (callee-saved) state of each call-frame and to resume execution at any point in the call-chain
+ (non-local goto). The API supports both local (same-process) and remote (across-process) operation.
+ As such, the API is useful in a number of applications"""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+dependencies = [ ('binutils', '2.25.1'), ]
+
+sources = [SOURCE_TAR_GZ]
+source_urls = [GNU_SAVANNAH_SOURCE]
+
```

Comment:
@jgphpc libunwind depends on XZ, see also https://github.com/geimer/easybuild-easyconfigs/pull/4 and https://github.com/hpcugent/easybuild-easyconfigs/pull/3055